src/dev/build/tasks/bin/scripts/opensearch-dashboards

```diff
@@ -15,8 +26,15 @@ done
 
 DIR="$(dirname "${SCRIPT}")/.."
 CONFIG_DIR=${OSD_PATH_CONF:-"$DIR/config"}
-NODE="${DIR}/node/bin/node"
+
+if [ $OS = "freebsd" ]; then
+  NODE="/usr/local/bin/node"
+else
+  NODE="${DIR}/node/bin/node"
+fi
+
```

Comment:
Another possibility (maybe simpler): if a bundled `node` is present, use it, otherwise expect one to bie found in $PATH:

```suggestion

if [ -x "${DIR}/node/bin/node" ]; then
  NODE="${DIR}/node/bin/node"
else
  NODE="node"
fi

```

Reply:
What about a
`elif [ ! -x "$(which node)" ]; then`
looks a little bit more solid then using `node` without a path. Not sure if has any side effects.

Reply:
That's probably better and allow less modifications of the rest of the script.

Reply:
@hackacad  were you able to get insight getting node without the path.

For me, i think the original code for me I can pretty explicitly read that we are doing this conditional for FreeBSD node which makes me feel safer.

Reply:
Using a more general condition looks better, but it should still look for a path. Using `node` without a path would cover most environments, but looking for it (like using which) should be much more solid.